packages/health-checks/src/health-checks.ts

```diff
@@ -47,7 +47,7 @@ export const HealthCheckConfig = {
   },
   responses: {
     contentType: 'application/health+json',
-    statusCodes: { healthy: 200, unhealthy: 503, unknown: 500 },
+    statusCodes: { healthy: 200, unhealthy: 503, timedout: 408 },
```

Comment:
I don't think a 408 is applicable here. Any 4xx status implies that the error occurred on the client (or because of the client), which will never be the case for a health check.

Reply:
Removed since timeout is no longer a valid overall status.